gadget/params.c

```diff
@@ -140,10 +140,11 @@ create_gadget_parameter_set()
     param_declare_double(ps, "Omega_fld", OPTIONAL, 0, "Energy density of dark energy fluid.");
     param_declare_double(ps, "w0_fld", OPTIONAL, -1., "Dark energy equation of state.");
     param_declare_double(ps, "wa_fld", OPTIONAL, 0, "Dark energy evolution parameter.");
+    param_declare_double(ps, "Omega_ur", OPTIONAL, 0, "Extra radiation density, eg, a sterile neutrino");
     param_declare_double(ps, "HubbleParam", REQUIRED, 0.697, "");
 
     param_declare_int(ps,    "OutputPotential", OPTIONAL, 1, "Save the potential in snapshots.");
-    param_declare_int(ps,    "MaxMemSizePerNode", OPTIONAL, 0.6 * get_physmem_bytes() / (1024 * 1024), "Preallocate this much memory MB per computing node/ host. Default is 80\% of total physical mem per node. ");
+    param_declare_double(ps,    "MaxMemSizePerNode", OPTIONAL, 0.6, "Preallocate this much memory per computing node/ host. If < 1, in fraction of total physical mem (default 0.6), otherwise in MB. ");
```

Comment:
rename it to fraction. I still think using bytes / MB is more reasonable because if one want to specify a number than the total available memory is already known. And usually it is Total - ReservedForOS. And also consider the case where we wrongly detected the total mem. A fraction we are totally stuck without workaround.

Reply:
This is not always a fraction: it is a fraction only if less than one. I will adjust the docstring.